hal_st/cortex/InterruptCortex.cpp

```diff
@@ -169,11 +174,14 @@ namespace hal
 
     void DispatchedInterruptHandler::Invoke()
     {
-        DisableInterrupt(Irq());
+        if (!Irq())
+            return;
```

Comment:
This if-statement should not be necessary: If this interrupt handler is not active, it cannot be invoked